chart/templates/import-api/secret.yaml

```diff
@@ -1,6 +1,7 @@
 {{ $secretContent := include "carto._utils.generateSecretObjects" (dict "vars" (list
   "ENCRYPTION_SECRET_KEY"
   "IMPORT_ACCESSKEYID"
+  "IMPORT_JWT_SECRET"
```

Comment:
This should be added also in a couple of additional places in the deployment (For all component/secrets)